tests/test_ssn_en_US.nim

```diff
@@ -0,0 +1,14 @@
+import std/unittest
+import faker
+import strscans
+
+test "SSN":
+  block:
+      checkpoint "it generates a valid ssn"
+      let fake = newFaker()
+      let ssn = fake.ssn()
+      let (success, aNum, gNum, sNum) = scanTuple(ssn, "$i-$i-$i")
```

Comment:
`scanTuple` may not exist in older Nim.
change procs:

```nim
import strutils

let x = "001-02-0003"
let a = x.split("-")
let aNum = a[0]
let gNum = a[1]
let sNum = a[2]

check aNum == 3
check gNum == 2
check sNum == 4
```

Reply:
I just was about to write that 😆

Reply:
Is there any type of array destructing in Nim?